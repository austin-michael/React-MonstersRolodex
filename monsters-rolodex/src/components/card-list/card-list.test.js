import { shallow } from "enzyme";
import CardList from "./card-list.component";

const mockMonsters = [
  {
    id: 1,
    name: "Leanne Graham",
    email: "Sincere@april.biz",
  },
  {
    id: 2,
    name: "Ervin Howell",
    email: "Shanna@melissa.tv",
  },
];

it("expect to render Card component", () => {
  expect(shallow(<CardList monsters={mockMonsters} />)).toMatchSnapshot();
});
