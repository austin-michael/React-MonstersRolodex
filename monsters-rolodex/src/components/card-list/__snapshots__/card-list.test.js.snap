// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card component 1`] = `
<div
  className="card-list"
>
  <Card
    key="1"
    monster={
      Object {
        "email": "Sincere@april.biz",
        "id": 1,
        "name": "Leanne Graham",
      }
    }
  />
  <Card
    key="2"
    monster={
      Object {
        "email": "Shanna@melissa.tv",
        "id": 2,
        "name": "Ervin Howell",
      }
    }
  />
</div>
`;
